src/GafferML/Inference.cpp

```diff
@@ -94,7 +119,18 @@ Ort::Session &acquireSession( const std::string &fileName )
 		throw Exception( fmt::format( "Could not find file \"{}\" on GAFFERML_MODEL_PATHS", fileName ) );
 	}
 
-	it = g_map.try_emplace( fileName, acquireEnv(), path.c_str(), Ort::SessionOptions() ).first;
+	auto sessionOpt = Ort::SessionOptions();
+	auto customLibraryPaths = customOpLibraryPaths();
+	for ( const auto& customLibraryPath : customLibraryPaths )
+	{
+		Ort::ThrowOnError(
+			Ort::GetApi().RegisterCustomOpsLibrary_V2(
```

Comment:
Can we use `sessionOpt.RegisterCustomOpsLibrary()` here to avoid dropping down to the C API?